README.md

```diff
@@ -42,6 +42,8 @@ rpcpassword=bar
 txindex=1
 zmqpubrawblock=tcp://127.0.0.1:29000
 zmqpubrawtx=tcp://127.0.0.1:29000
+
+deprecatedrpc=addwitnessaddress   # only needed with Bitcoin Core 0.16+
```

Comment:
You also need to specify `addresstype=p2sh-segwit` (we don't support bech32 addresses yet).
The sentence below about UTXO management should also be amended: with version 0.16 it is no longer necessary to specify change addresses manually.

Reply:
According to the Bitcoin Core 0.16.0 release notes, the addresstype "p2sh-segwit" is the default, bech32 is not enforced. 
https://bitcoin.org/en/release/v0.16.0#segwit-wallet

I think the whole paragraph "On testnet..." should be marked for Bitcoin Core 0.15 and earlier, like "On testnet with Bitcoin Core 0.15.1 and earlier...". And a new paragraph for Bitcoin Core 0.16+ should be added. I will test this thoroughly (wiping my wallet and creating a new, 0.16-native wallet) to describe this.

Reply:
`p2sh-segwit` is the default setting but if people change it eclair will not work properly, explicitly adding this option to `bitcoin.conf` makes it clearer.

Reply:
Following your explanation, I added the `addresstype=p2sh-segwit`.

I specified more clearly that the special handling of UTXO as `p2sh-of-p2wpkh` is needed in wallets created with Bitcoin Core 0.15.1 and earlier, as even a wallet migrated to 0.16.0 could contain `legacy` UTXO.

I tested setting up a newly created wallets in 0.16.0 and everything went as expected without any special UTXO handling.